src/test/java/org/folio/search/service/metadata/LocalSearchFieldProviderTest.java

```diff
@@ -85,6 +87,18 @@ void shouldAddStarNotationForMultilangField() {
       .containsExactly("title2.sub3.sub5.*");
   }
 
+  @Test
+  void shouldReturnAllExpectedFieldsForInstanceTitleSearch() {
```

Comment:
this test looks like a redundant:
1. It depends of existing model, which can be modified
2. Behavior of inventory search types already tested in other unit tests

Reply:
This test has been removed.